src/nbl/video/IOpenGL_PhysicalDeviceBase.h

```diff
@@ -674,6 +676,8 @@ class IOpenGL_PhysicalDeviceBase : public IOpenGLPhysicalDeviceBase
 			m_properties.limits.maxBufferViewSizeTexels = m_glfeatures.maxTBOSizeInTexels;
 			m_properties.limits.maxUBOSize = m_glfeatures.maxUBOSize;
 			m_properties.limits.maxSSBOSize = m_glfeatures.maxSSBOSize;
+
+			m_properties.limits.maxPushConstantsSize = 128u;
```

Comment:
dont write out a constant literal use `IGPUMeshBuffer::MAX_` thingie

Reply:
Thought about it, was scared of circular inclusion or something

Reply:
then use `asset::ICPUMeshBuffer::MAX_PUSH_CONSTANT_BYTESIZE`